dockstore-webservice/src/main/resources/openapi.yaml

```diff
@@ -4843,6 +4903,11 @@ components:
       name: Authorization
       in: header
   schemas:
+    Alias:
+      type: object
+      properties:
+        content:
+          type: string
```

Comment:
Is there a plan on adding more properties to this?

Reply:
I believe so